pages/view_case/index.tsx

```
@@ -15,40 +15,23 @@ const ViewCase = () => {
-            if (!response.ok) {
-                throw new Error(`HTTP error! Status: ${response.status}`);
+    useEffect(() => {
+        if (!caseNumber.trim()) return;
+
+        const fetchCaseSummary = async () => {
+            setLoading(true);
+            try {
+                const response = await fetch(`/api/case/${year}/${caseType}/${caseNumber}`);
+                const res = await response.json();
+                setData(res.caseSummary || []);
+            } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)